Jenkinsfile_CNP

```diff
@@ -34,6 +34,17 @@ def secrets = [
         secret('ccd-restricted-cross-case-type-caseworker-email', 'CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_EMAIL'),
         secret('ccd-restricted-cross-case-type-caseworker-password', 'CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_PASSWORD'),
         secret('ccd-api-gateway-oauth2-client-secret', 'OAUTH2_CLIENT_SECRET'),
+
+        //Required for FTA test RDM-6943
+        secret('ccd-caseworker-solicitor-one-password', 'CCD_BEFTA_CASEWORKER_2_SOLICITOR_1_PWD'),
```

Comment:
Since we have renamed the values of the secret to add **_BEFTA_** we should rename the key values too to reflect this like the other  secret key values eg:
ccd-caseworker-solicitor-one-password -> ccd-**befta-**caseworker-solicitor-one-password

Reply:
@RebeccaBaker  - Could we pls rename the secrets to the same name as that of the environment variable, except for a "-" instead of "_"?